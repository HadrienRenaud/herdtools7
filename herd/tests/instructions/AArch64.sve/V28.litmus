AArch64 V28
{
uint8_t t[6] = {255,255,255,255,255,255};
0:X0=t;
}
  P0                    ;
  INDEX Z1.B,#1,#3      ;
  PTRUE P0.B,VL6        ;
  LD1B {Z0.B},P0/Z,[X0] ;
  ADD Z0.B,Z0.B,Z1.B    ;
  ST1B {Z0.B},P0,[X0] ;

forall t={0,3,6,9,12,15}
