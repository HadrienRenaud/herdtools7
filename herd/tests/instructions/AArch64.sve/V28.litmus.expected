Test V28 Required
States 1
t={0,3,6,9,12,15};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (t={0,3,6,9,12,15})
Observation V28 Always 1 0
Hash=bc74fed293e5506ea70cae45c5fb0911

